src/pineko/evolve.py

```diff
@@ -269,6 +270,8 @@ def evolve_grid(
     rich.print(f"Optimizing for {assumptions}")
     fktable.optimize(assumptions)
     fktable.set_key_value("eko_version", operators.metadata.version)
+    fktable.set_key_value("eko_theory", json.dumps(operators.theory_card.raw))
+    fktable.set_key_value("eko_operator", json.dumps(operators.operator_card.raw))
```

Comment:
Is there a particular reason to call it `eko_operator`? Because I find rather confusing given that we are actually talking about the operator card. I would rather call it `eko_operator_card` or `eko_opcard`.

Reply:
I was first thinking the same, but then I thought "its obvious" - turns out it is not, we can just change -done in [697af5f](https://github.com/NNPDF/pineko/pull/171/commits/697af5f40ab3eee4e93d83abcb9c5058258e0986)